SECURITY.md

```diff
@@ -0,0 +1,46 @@
+
+We appreciate your efforts to disclose any security findings responsibly, and we are committed to collaborating with you to address the issue. Rest assured that you will be given proper credit for your contribution.
+
+## Responsible Disclosure
+
+Please practice responsible disclosure when reporting security issues. We kindly request that you do not publicly disclose or discuss the vulnerability until we have had sufficient time to investigate and address it. Privately disclosing the issue to us via email allows us to work on a fix without putting our users' funds at risk.
+
+We understand the importance of acknowledging and rewarding security researchers for their efforts. Even though our bug bounty program is currently set up, we assure you that you will not be unpaid for your valuable contributions. We are committed to fairly compensating those who help us improve the security of our platform.
+
+## Security Reviews and Audits
+
+At Inverter Network, we prioritize the security of our smart contracts and follow a rigorous process of security reviews and audits:
+
+* Per our development and security guidelines, we require an external review for each pull request (PR) that goes from the `dev` branch to the `main` branch in our repository. We have partnered with [Team Omega](https://teamomega.eth.limo) to conduct these security reviews directly in the PRs.
+
+* Additionally, prior to major releases, we perform a full audit of any changed contracts via an external auditing company before the deployment. This ensures that our contracts undergo thorough scrutiny by professional auditors.
+
+* Our version 1 has been audited by [0xMacro](https://0xmacro.com/). The audit report will be published here once it becomes public. Furthermore, our v1 is currently undergoing a public audit competition on [Hats Finance](https://hats.finance/), leveraging the expertise of the broader security community.
+
+We are committed to transparency and will make the results of our security reviews and audits available to the public, fostering trust and confidence in the security of our platform.
+
+## Security Guideline
+
+At Inverter Network, we adhere to a comprehensive Security Guideline that outlines our best practices and approach to ensuring the security of our smart contracts. This guideline is a public document that we follow at all times. You can find our Security Guideline [here](https://docs.google.com/document/d/1CZgM9OEuibNrimbNeActve5n9ro3Ydu03OfSnZfRo_s).
+
+We encourage you to review our Security Guideline to understand our commitment to security and the measures we have in place to protect our users and the integrity of our platform.
+
+## Bug Bounty Program
```

Comment:
Maybe we can move the bug bounty part a bit more upwards, closer to the actual reporting header. In this way we can leave out 1 of the 2 mentions of "You will be compensated" within the sections _Reporting Security Issues_ & _Responsible Disclosure_, as readers will know to read on to the Bug Bounty sections

Reply:
Yes, this will be updated once we have something, as discussed today. Will leave it like this for now.